src/main/java/com/dokev/gold_dduck/gift/GiftItem.java

```diff
@@ -22,6 +22,9 @@
     @Column(name = "content", columnDefinition = "TEXT", nullable = false)
     private String content;
 
+    @Column(name = "used", nullable = false)
+    private Boolean used;
```

Comment:
```suggestion
    private boolean used;
```
notnull에 기본값 false가 기본이니깐
primitive 타입으로 바꾸면 좋을 것 같습니다!